lib/flutter/flutter_connect.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:atom/atom.dart';
+import 'package:atom/utils/disposable.dart';
+import 'package:logging/logging.dart';
+
+import '../elements.dart';
+import '../launch/launch_configs.dart';
+import '../projects.dart';
+import '../state.dart';
+import 'flutter_daemon.dart';
+import 'flutter_devices.dart';
+import 'flutter_launch.dart';
+
+final Logger _logger = new Logger('flutter.connect');
+
+/// Connect the tools to a Flutter app that is already running on a device.
+class FlutterConnectManager implements Disposable {
+  Disposables _disposables = new Disposables();
+
+  ConnectDialog connectDialog;
+
+  FlutterConnectManager() {
+  }
+
+  void showConnectDialog() {
+    if (connectDialog == null) {
+      connectDialog = new ConnectDialog();
+      _disposables.add(connectDialog);
+    }
+    connectDialog.show();
+  }
+
+  void dispose() => _disposables.dispose();
+}
+
+class ConnectDialog implements Disposable {
+  TitledModelDialog dialog;
+  CoreElement _listGroup;
+
+  ConnectDialog() {
+    dialog = new TitledModelDialog('Connect to Flutter App', classes: 'jobs-dialog');
```

Comment:
We probably want to change the css name (and its uses) from `jobs-dialog` to something more general like `list-dialog`. Can you add a TODO? (I'll follow it up)